From 232fbf25c9aef7e84fa9876f52bbf4bb1fb5ffdc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Wed, 18 Aug 2010 14:31:38 +0000
Subject: [PATCH] tests: fix syntax error in "Use advise() for hints" test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the test introduced in the "Use advise() for hints" patch by
Jonathan Nieder not to use '' for quotes inside '' delimited code. It
ended up introducing a file called <paths> to the main git repository.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t3507-cherry-pick-conflict.sh |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/t/t3507-cherry-pick-conflict.sh b/t/t3507-cherry-pick-conflict.sh
index 3f29594..607bf25 100755
--- a/t/t3507-cherry-pick-conflict.sh
+++ b/t/t3507-cherry-pick-conflict.sh
@@ -38,7 +38,7 @@ test_expect_success 'failed cherry-pick does not advance HEAD' '
 	test "$head" = "$newhead"
 '
 
-test_expect_success 'advice from failed cherry-pick' '
+test_expect_success 'advice from failed cherry-pick' "
 	git checkout -f initial^0 &&
 	git read-tree -u --reset HEAD &&
 	git clean -d -f -f -q -x &&
@@ -46,17 +46,17 @@ test_expect_success 'advice from failed cherry-pick' '
 	git update-index --refresh &&
 	git diff-index --exit-code HEAD &&
 
-	picked=$(git rev-parse --short picked) &&
+	picked=\$(git rev-parse --short picked) &&
 	cat <<-EOF >expected &&
-	error: could not apply $picked... picked
+	error: could not apply \$picked... picked
 	hint: after resolving the conflicts, mark the corrected paths
 	hint: with 'git add <paths>' or 'git rm <paths>'
-	hint: and commit the result with 'git commit -c $picked'
+	hint: and commit the result with 'git commit -c \$picked'
 	EOF
 	test_must_fail git cherry-pick picked 2>actual &&
 
 	test_cmp expected actual
-'
+"
 
 test_expect_success 'failed cherry-pick produces dirty index' '
 
-- 
1.7.2.1.414.g9bf49

